src/predicates.js

```diff
@@ -19,6 +24,10 @@ const FILTER_OPERATORS = {
   kv_equal: function(obj, params) { return obj[params[0]] === params[1]; },
   kv_not_contains: function(obj, params) { return !((params[0] in obj) && (obj[params[0]].indexOf(params[1]) >= 0)); },
   kv_not_equal: function(obj, params) { return obj[params[0]] !== params[1]; },
+  less_than: function(a, b) { return a < b; },
+  less_than_or_equal_to: function(a, b) { return a <= b; },
+  typeless_equal: function(a, b) { return a == b; },
+  typeless_not_equal: function(a, b) { return a != b; },
```

Comment:
MDN uses the terms "strict equality" and "loose equality"
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness
```suggestion
  loose_equal: function(a, b) { return a == b; },
  loose_not_equal: function(a, b) { return a != b; },
```

Reply:
Thanks, updated